src/policy/policy.cpp

```diff
@@ -75,7 +75,9 @@ bool IsStandard(const CScript& scriptPubKey, txnouttype& whichType)
 
 bool IsStandardTx(const CTransaction& tx, bool permit_bare_multisig, const CFeeRate& dust_relay_fee, std::string& reason)
 {
-    if (tx.nVersion > CTransaction::MAX_STANDARD_VERSION || tx.nVersion < 1) {
+    if (tx.nVersion > CTransaction::MAX_STANDARD_VERSION || tx.nVersion < 1 ||
```

Comment:
nit: `tx.nVersion == 0 || tx.nVersion > CTransaction::MAX_STANDARD_VERSION` is easier to read.
The check on `0x80000000` is superfluous as it is always bigger than `MAX_STANDARD_VERSION`, and I don't see any alternate dimension where this can be different.

Reply:
Actually you can just keep what was there before...

Reply:
Oops, duh.